.github/workflows/_test_pax_rosetta.yaml

```diff
@@ -98,7 +98,6 @@ jobs:
           #SBATCH --gpus-per-node=${{ steps.meta.outputs.GPUS_PER_NODE }}
           #SBATCH --time=00:30:00
           #SBATCH --output=${{ steps.meta.outputs.LOG_FILE }}
-          #SBATCH --export="VOCAB_PATH=gs://t5-data/vocabs/cc_all.32000.100extra/sentencepiece.model,ENROOT_PASSWORD=${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I think `ENROOT_PASSWORD=${{ secrets.GITHUB_TOKEN }}` should still be set

Reply:
Good catch. I'll fix that